src/app/pages/ArticlePage/Byline/index.tsx

```diff
@@ -116,11 +119,16 @@ const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
           <li
             css={
               isRtl
-                ? [BylineCss.imageRtl, BylineCss.Image]
-                : [BylineCss.imageLtr, BylineCss.Image]
+                ? [BylineCss.imageRtl, BylineCss.ImageWrapper]
+                : [BylineCss.imageLtr, BylineCss.ImageWrapper]
             }
           >
-            <img css={BylineCss.imageSource} src={image} alt="" />
+            <Image
+              css={BylineCss.imageSrc}
+              src={image}
```

Comment:
I think the `src` may need some URL construct applied to it to point to the correct iChef location.

Looking at the data, the `locator` field will only have part of the URL, e.g: `"locator": "f2f8/test/49056060-fea9-11ec-8adf-7bf4017ee9f9.png",` so I think you may need to use the `buildIChefUrl` function found in [#lib/utilities/ichefURL](https://github.com/bbc/simorgh/blob/7e36689744a7d29d286293a887669ec49d8f7db4/src/app/lib/utilities/ichefURL/index.js#L20)